environment.yml

```diff
@@ -3,7 +3,7 @@ channels:
     - conda-forge
     - nodefaults
 dependencies:
-    - python=3.12
+    - python>=3.10
```

Comment:
Note that this has changed from `python=3.12` to `python>=3.10` to folllow SPEC 0 (same with NumPy, pandas and xarray lines below).